spec/javascripts/modules/gtm-reorder-listener-spec.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-env jasmine */
+
+describe('Gtm reorder listener', function () {
+  'use strict'
+
+  var container
+
+  describe('when data-gtm attribute is specified', function () {
+    beforeEach(function () {
+      container = document.createElement('div')
+      container.dataset.gtm = 'reorderable-list-drag'
+      container.innerHTML =
+        '<button type="button" data-gtm="reorderable-list-move-up">Up</button>' +
+        '<button type="button" data-gtm="reorderable-list-move-down">Down</button>'
+
+      document.body.appendChild(container)
+      window.dataLayer = []
+    })
+
+    afterEach(function () {
+      document.body.removeChild(container)
+    })
+
+    describe('reorder-move-up events', function () {
```

Comment:
Minor: could we combine these two (up/down) into a single spec? We do this [for copy/paste](https://github.com/alphagov/content-publisher/blob/master/spec/javascripts/modules/gtm-copy-paste-listener-spec.js), as the events are so similar.

Reply:
Combined.